Templates/colors.json

```diff
@@ -0,0 +1,4687 @@
+        { "name": "Unknown200", "type": "uint32" },
+        { "name": "Unknown201", "type": "uint32" },
+        { "name": "Unknown202", "type": "uint32" },
+        { "name": "Unknown203", "type": "uint32" },
+        { "name": "Unknown204", "type": "uint32" },
+        { "name": "Unknown205", "type": "uint32" },
+        { "name": "Unknown206", "type": "uint32" },
+        { "name": "Unknown207", "type": "uint32" },
+        { "name": "Unknown208", "type": "uint32" },
+        { "name": "Unknown209", "type": "uint32" },
+        { "name": "Unknown210", "type": "uint32" },
+        { "name": "Unknown211", "type": "uint32" },
+        { "name": "Unknown212", "type": "uint32" },
+        { "name": "Unknown213", "type": "uint32" },
+        { "name": "Unknown214", "type": "uint32" },
+        { "name": "Unknown215", "type": "uint32" },
+        { "name": "Unknown216", "type": "uint32" },
+        { "name": "Unknown217", "type": "uint32" },
+        { "name": "Unknown218", "type": "uint32" },
+        { "name": "Unknown219", "type": "uint32" },
+        { "name": "Unknown220", "type": "uint32" },
+        { "name": "Unknown221", "type": "float32" },
+        { "name": "Unknown222", "type": "float32" },
+        { "name": "Unknown223", "type": "float32" },
+        { "name": "Unknown224", "type": "float32" },
+        { "name": "Unknown225", "type": "float32" },
+        { "name": "Unknown226", "type": "uint32" },
+        { "name": "Unknown227", "type": "float32" },
+        { "name": "Unknown228", "type": "uint32" },
+        { "name": "Unknown229", "type": "uint32" },
+        { "name": "Unknown230", "type": "float32" },
+        { "name": "Unknown231", "type": "float32" },
+        { "name": "Unknown232", "type": "float32" },
+        { "name": "Unknown233", "type": "uint32" },
+        { "name": "Unknown234", "type": "float32" },
+        { "name": "Unknown235", "type": "uint32" },
+        { "name": "Unknown236", "type": "uint32" },
+        { "name": "Unknown237", "type": "float32" },
+        { "name": "Unknown238", "type": "float32" },
+        { "name": "Unknown239", "type": "uint32" },
+        { "name": "Unknown240", "type": "float32" },
+        { "name": "Unknown241", "type": "float32" },
+        { "name": "Unknown242", "type": "float32" },
+        { "name": "Unknown243", "type": "float32" },
+        { "name": "Unknown244", "type": "float32" },
+        { "name": "Unknown245", "type": "float32" },
+        { "name": "Unknown246", "type": "float32" },
+        { "name": "Unknown247", "type": "float32" },
+        { "name": "Unknown248", "type": "uint32" },
+        { "name": "Unknown249", "type": "uint32" },
+        { "name": "Unknown250", "type": "float32" },
+        { "name": "Unknown251", "type": "float32" },
+        { "name": "Unknown252", "type": "float32" },
+        { "name": "Unknown253", "type": "float32" },
+        { "name": "Unknown254", "type": "uint32" },
+        { "name": "Unknown255", "type": "uint32" }
```

Comment:
No parameter names were given because this is absolutely ridiculous what is this Sonic Team.